autogen.sh

```diff
@@ -1,2 +1,6 @@
-#!/bin/sh
-autoreconf -i
+#!/bin/bash -e
+
+  # (Re)Generate autotools files
```

Comment:
How about removing the leadnig whitespace here?

Reply:
I like to add an indentation after the header for all the commands subject to that header, but that's purely personal so I will remove it.